Tests/xcprojTests/PBXProjWriterSpec.swift

```diff
@@ -49,29 +51,30 @@ class PBXProjEncoderSpec: XCTestCase {
             "$": "$".quoted,
             "{": "{".quoted,
             "}": "}".quoted,
-            "\\": "\\".quoted,
+            "\\": escapedEscape.quoted,
             "+": "+".quoted,
             "-": "-".quoted,
             "=": "=".quoted,
             ",": ",".quoted,
             " ": " ".quoted,
+            "\t": escapedTab.quoted,
             "a;": "a;".quoted,
             "a_a": "a_a",
             "a a": "a a".quoted,
             "": "".quoted,
             "a\(quote)q\(quote)a": "a\(escapedQuote)q\(escapedQuote)a".quoted,
-            "a\(quote)q\(quote)a".quoted: "a\(escapedQuote)q\(escapedQuote)a".quoted,
-            "a\(escapedQuote)a\(escapedQuote)": "a\(escapedQuote)a\(escapedQuote)".quoted,
+            "a\(quote)q\(quote)a".quoted: "\(escapedQuote)a\(escapedQuote)q\(escapedQuote)a\(escapedQuote)".quoted,
+            "a\(escapedQuote)a\(escapedQuote)": "a\(escapedEscape)\(escapedQuote)a\(escapedEscape)\(escapedQuote)".quoted,
             "a\na": "a\\na".quoted,
             "\n": escapedNewline.quoted,
             "\na": "\(escapedNewline)a".quoted,
             "a\n": "a\(escapedNewline)".quoted,
-            "a\na".quoted: "a\(escapedNewline)a".quoted,
-            "a\(escapedNewline)a": "a\(escapedNewline)a".quoted,
-            "a\(escapedNewline)a".quoted: "a\(escapedNewline)a".quoted,
+            "a\na".quoted: "\(escapedQuote)a\(escapedNewline)a\(escapedQuote)".quoted,
+            "a\(escapedNewline)a": "a\(escapedEscape)na".quoted,
```

Comment:
Here’s an example of an already escaped character getting escaped again

Reply:
This covers case where user has typed 
`a\na` 
not
```
a
a
```

Example: https://github.com/RepositoryFork/xcproj/blob/d8a6995c23f8a5bd2f0aea68cb6461e7f08ee788/Fixtures/iOS/BuildSettings.xcodeproj/project.pbxproj#L100